src/server/index.jsx

```diff
@@ -279,12 +280,6 @@ server.get(
         });
       }
 
-      logger.info(ROUTING_INFORMATION, {
-        url,
-        status,
```

Comment:
By removing this, we no longer have the status being logged... is this acceptable?

Reply:
_Also_ I think we may want to still log as info (or indeed warning?) where the status is non-200 or our error queries will no longer function at all!